typescript/packages/common-memory/store.ts

```diff
@@ -406,3 +422,34 @@ export const query = (
     return { error: Error.query({ the, of, in: id }, error as SqliteError) };
   }
 };
+
+export const list = (
+  { id, store }: Model,
+  the: string
+): Result<ListResult[], ToJSON<ListError>> => {
+  try {
+    const LIST_QUERY = `
+      SELECT
+        state.of as 'of',
+        state."is" as 'is'
+      FROM state
+      WHERE state.the = :the
+    `;
+
+    const rows = store.prepare(LIST_QUERY).all({ the }) as Array<{
+      of: Entity;
+      is: string | null;
```

Comment:
If `is` set to `null` it implies that fact was retracted a.k.a document was deleted, I don't know if listing those in the results makes much sense. Unless getting such records are intentional I would suggest modifying `where` clause to exclude records where `is` is `NULL`